app/parsers/bulkrax/csv_parser.rb

```diff
@@ -85,7 +85,7 @@ def valid_import?
     end
 
     def create_collections
-      collections.each_with_index do |collection, index|
+      collections.each_with_index do |collection|
```

Comment:
```suggestion
      collections.each do |collection|
```

Reply:
changes were reverted so this change no longer applies.